modules/gapi/src/backends/common/gbackend.hpp

```diff
@@ -14,7 +14,7 @@
 #include <ade/node.hpp>
 
 #include "opencv2/gapi/garg.hpp"
-#include "opencv2/gapi/own/mat.hpp"
+// #include "opencv2/gapi/own/mat.hpp"
```

Comment:
Delete?

Reply:
Last commit doesn't contain this `include"opencv2/gapi/own/mat.hpp"`.

Reply:
Thanks